lib/Edited/prototype/listen.js

```diff
@@ -1,18 +1,42 @@
 'use strict'
-var on
-var TEXT_ALTERING_EVENT_TYPES
-
-on = require('add-event-handler')
-TEXT_ALTERING_EVENT_TYPES = require('../../text-altering-event-types')
+var on = require('add-event-handler')
+var Combokeys = require('combokeys')
 
 module.exports = function () {
   var self = this
 
-  self.listener = function (e) {
-    self.discern(e)
+  // a combokeys instance for character addition
+  self.anyCharCombokeys = new Combokeys(self.element)
+  // `any-character` is a combokeys special argument that
+  // calls back on any character
+  self.anyCharCombokeys.bind('any-character', function () {
+    self.discern('character-addition')
+  })
+  self.anyCharCombokeys.stopCallback = function (e) {
+    if (e.charCode === 32) {
```

Comment:
Is there any way we could use a named constant for the `32`? `SPACEBAR` or some such? Then, you don't need the comment because the code will be self-documenting.